third_party/xla_client/xrt_computation_client.h

```diff
@@ -114,22 +114,32 @@ class XrtComputationClient : public ComputationClient {
       return task_no == rhs.task_no && name == rhs.name;
     }
 
+    std::string ToString() const {
+      return name + ":" + std::to_string(task_no);
+    }
+
+    friend std::ostream& operator<<(std::ostream& stream,
+                                    const Worker& worker) {
+      stream << worker.name << ":" << worker.task_no;
+      return stream;
+    }
+
     std::string name;
     int task_no;
   };
 
   struct Options {
     std::string default_device;
-    // Maps a PyTorch device ID (example, "GPU:0", "TPU:0") to the full
+    // Maps a PyTorch Worker (example, {"GPU", 0}, {"TPU", 0}) to the full
     // coordinates in TF device format
     // (ie, /job:tpu_worker/replica:0/task:0/device:TPU:0), of the worker
     // exposing that device. These devices are all the devices present within
     // the TPU mesh.
-    std::map<std::string, std::string> global_device_map;
+    std::map<Worker, std::string> global_device_map;
     // These are the devices that this instance of PyTorch is handling. These
     // devices are in the form of "CPU:0", "TPU:3", ... For each of these
     // devices, there is an entry within the global_device_map.
-    std::set<std::string> devices;
+    std::set<string> devices;
```

Comment:
nit, let's keep it as `std::string` since the rest of the string in this file come with the explicit std namespace.

Reply:
Oops, didn't mean to change that.